Implement_A_Stack/myDataStructs.py

```diff
@@ -1,102 +1,94 @@
 # -*- coding: utf-8 -*-
-"""
-Spyder Editor
-
-This is a temporary script file.
-"""
 import copy
 
-class myStack(object):
-    'My stack implementation'
+class MyStack(object):
+    """My stack implementation"""
+
     def __init__(self):
-        self.contents = ()
+        self.__contents = ()
 
 
     def empty(self):
-        if len(self.contents):
-            return False
-        else:
-            return True
+        return len(self.__contents) == 0
 
 
     def push(self, val):
-        self.contents = (val, self.contents)
+        self.__contents = (val, self.__contents)
 
 
     def pop(self):
```

Comment:
Return None as sentinel value and confuse client code, or raise exception and clutter client code?